src/include/bootloader/routines/autoinstall.rb

```diff
@@ -130,6 +130,7 @@ def AI2Export(ai)
         # FIXME: this should be better handled by exception and show it properly, but it require too big change now
         Popup.Error(_("Unsupported bootloader '%s'. Adapt your AutoYaST profile accordingly.") %
  exp["loader_type"])
+        Builtins.y2error("unsupported bootloader #{exp["loader_type"]}")
```

Comment:
autoinstall code do not yet use new logging system and to be honest I am quite scared to touch almost anything in autoyast part of bootloader as it is full of workarounds and backward compatibility :)